tests/tas-elasticsearch/src/test/resources/test-suites/elasticsearch-suite.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
 <suite name="REST API tests Elasticsearch" preserve-order="true" verbose="1" parallel="false">
-  <test name="elasticsearch" verbose="3">
+  <listeners>
+    <listener class-name="org.alfresco.elasticsearch.AlphabeticallyPriorityInterceptor" />
```

Comment:
Related to the comment above, should reference the listener inherited via the comm-repo test sources.